cmapi/cmapi_server/__main__.py

```diff
@@ -224,6 +167,10 @@ def create_self_signed_certificate():
     # subscribe FailoverBackgroundThread plugin code to bus channels
     # code below not starting "real" failover background thread
     FailoverBackgroundThread(cherrypy.engine, turn_on_failover).subscribe()
+    cherrypy.engine.certificate_monitor = plugins.BackgroundTask(
+        3600, CertificateManager.renew_certificate
```

Comment:
Raise the number up to 86400. Once a day should be enough.

Reply:
Checking if certificate will expire in 1 day, so setting 86400 could be potentially harmful.